.taskcluster.yml

```diff
@@ -0,0 +1,687 @@
+allowPullRequests: collaborators
+tasks:
+- extra:
+    github:
+      branches: [master]
+      events: [push]
+  metadata: {description: '', name: wpt-chrome-dev-reftest-1, owner: '{{ event.head.user.email
+      }}', source: '{{ event.head.repo.url }}'}
+  payload:
+    artifacts:
+      public/results: {path: /home/test/artifacts, type: directory}
+    command: [/bin/bash, --login, -c, ">-\n            ~/start.sh &&\n           \
+        \ cd /home/test/web-platform-tests &&\n            git fetch {{event.head.repo.url}}\
+        \ &&\n            git config advice.detachedHead false &&\n            git\
+        \ checkout {{event.head.sha}} &&\n            ./tools/ci/ci_taskcluster.sh\
+        \ chrome reftest 1 6"]
+    image: harjgam/web-platform-tests:0.5
```

Comment:
Should we have a "web-platform-tests" organization on Docker Hub? Is a thing?

Reply:
It is and we can; I just created an org with the namespace `webplatformtests`; neither `wpt` or `web-platform-tests` worked, but I can't tell if they were taken or if there are some character/length restrictions.

Reply:
Where does the code live for creating the docker images?